generators/client/templates/angular/_tslint.json

```diff
@@ -124,9 +124,6 @@
         "use-life-cycle-interface": true,
         "use-pipe-transform-interface": false,
         "component-class-suffix": true,
-        "directive-class-suffix": true,
-        "no-access-missing-member": true,
```

Comment:
are these props deprecated?

Reply:
check the comments over here 
https://github.com/jhipster/ng-jhipster/pull/53#discussion_r149281262